### DISCUSSION TITLE
Remix-i18next doesn't support arrays as fallback languages

### DISCUSSION THREAD

**Discussion by Loschcode** [sergiodxa/remix-i18next#187] [Q&A]
When using `remix-i18next` I add the following

```
export default {
  supportedLngs: ["es-MX", "en-US"],
  fallbackLng: {
    "es-MX": ["es", "en-US"],
    default: ["en-US"],
  },
};
```

I then use that configuration on the `i18next.server.tsx`

```
const i18next = new RemixI18Next({
  detection: {
    supportedLanguages: i18n.supportedLngs,
    fallbackLanguage: i18n.fallbackLng,
  },
```

However, it doesn't seem to be supported, `fallbackLanguage` will yield an error and complain that it's not a `string` and of course it won't complain and work perfectly when using `en-US` as simple `fallbackLanguage`

This is quite weird for me since i18next [let you do just that](https://www.i18next.com/principles/fallback) out of the box. It's essential if you want to support multiple different Spanish or English versions for example and want a fallback in what you want to be the most neutral for your project.

I don't think it's a bug, it may be on purpose, but why is that? Or maybe there's something I didn't quite figure out.

**Comment by sergiodxa** [comment]
Because:

1. I didn't know i18next could do that
2. Those options while similar to i18next, are used by the remix-i18next LanguageDetector class that works fully server-side and doesn't really depend on i18next.
